app/src/types/generic/endpoints.ts

```diff
@@ -18,4 +18,5 @@ export enum endpoints {
     UPLOAD_DOCUMENT_STATE = '/UploadState',
 
     ODS_REPORT = '/OdsReport',
+    MOCK_LOGIN = '/login',
```

Comment:
To keep the endpoint naming inline with the other authentication endpoints, I suggest modifying it like so:
```suggestion
    MOCK_LOGIN = '/Auth/MockLogin',
```